src/Illuminate/Broadcasting/BroadcastManager.php

```diff
@@ -74,14 +74,42 @@ public function routes(array $attributes = null)
                 ['get', 'post'], '/broadcasting/auth',
                 '\\'.BroadcastController::class.'@authenticate'
             )->withoutMiddleware([\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken::class]);
+        });
+    }
+
+    /**
+     * Register the routes for handling broadcast authentication.
+     *
+     * @param  array|null  $attributes
+     * @return void
+     */
+    public function userAuthenticationRoutes(array $attributes = null)
```

Comment:
Probably best to just make this an empty array fallback rather than use two different types.